pyproject.toml

```diff
@@ -64,6 +64,28 @@ select = [
     "PLE", #pylint
 ]
 
+extend-select = [
+    "F401",    # unused-import
+    "F841",    # Unused variable
+    "W291",    # Trailing whitespace (low-hanging fruit that improves cleanliness)
+    "D419",    # Empty docstring (suggests bad documentation hygiene.)
+    "PLW0127", # Self-assigning variable
+    "PLW0128", # Redeclared assigned name
+    "UP004",   # Useless object inheritance
+    "PLE0101", # return in __init__ – breaks instantiation logic.
+    "PLE0704", # Misplaced bare raise – makes debugging harder.
+    "PLE1142", # Await outside async function – common async error.
+    "PLR1711", # Useless return - stylistic but can be misleading.
+    "PLR0124", # Comparison with itself – almost always a bug, but rare.
+    "PLW3301", # Nested min/max – niche readability case.
+    "PLR1702", # Too many nested blocks
+    "PLE0604", # Ensures __all__ is correctly defined.
+    "PLE1300", # Catches invalid string format chars.
+    "PLE0241", # duplicate-bases – can cause runtime errors.
+    "F702",    # not-in-loop
+    "PLW0133", # pointless-exception-statement
```

Comment:
Please sort this alphabetically so that codes within the same group are next to each other (so that all `PLW` ones are one after the other and so on)